ui/lib/kmip/addon/components/page/credentials-generate.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import errorMessage from 'vault/utils/error-message';
+
+export default class KmipPageCredentialsGenerate extends Component {
+  @service flashMessages;
+  @service('app-router') router;
+  @tracked hasGenerated = false;
+
+  @action
+  async revokeCredentials() {
+    const { scope, role } = this.args.credentials;
+    try {
+      await this.args.credentials.destroyRecord();
+      this.flashMessages.success('Successfully revoked credentials');
```

Comment:
Nit: missing period at the end. We’re not consistent here, but if it’s a full user-facing sentence (and not a title or heading), I think it should end with a period.

Reply:
These were copy and pasted from the original messages, but I can add